src/ray/core_worker/store_provider/memory_store_provider.cc

```diff
@@ -49,8 +49,11 @@ Status CoreWorkerMemoryStoreProvider::Wait(
   std::vector<ObjectID> id_vector(object_ids.begin(), object_ids.end());
   std::vector<std::shared_ptr<RayObject>> result_objects;
   RAY_CHECK(object_ids.size() == id_vector.size());
-  RAY_RETURN_NOT_OK(
-      store_->Get(id_vector, num_objects, timeout_ms, false, &result_objects));
+  auto status = store_->Get(id_vector, num_objects, timeout_ms, false, &result_objects);
+  // Absorb TimedOut statuses.
```

Comment:
```suggestion
  // Ignore TimedOut statuses.
```

Please comment why they're ignored. Also, if we just ignore them should they return timed out at all?

Reply:
Sure, i'll elaborate in the comment. 

By "they" do you mean `memory_store::Get`? For `CoreWorkerMemoryStoreProvider::Get` this is the correct behavior. But since `Wait` uses the same `memory_store::Get` function, it has to account for that. It doesn't make sense to pull the `TimedOut` status return out to `CoreWorkerMemoryStoreProvider::Get` since it's hard to discern whether or not the objects are `null` because of a timeout or because they're actually `null`. Or at least that's my understanding.